aten/src/ATen/cudnn/Handle.cpp

```diff
@@ -12,8 +13,10 @@ namespace {
 
 struct Handle {
   cudnnHandle_t handle;
-  Handle() : handle(nullptr) {
-    AT_CUDNN_CHECK(cudnnCreate(&handle));
+  Handle(bool create = false) : handle(nullptr)
```

Comment:
nit: You could make this more explicit by using the named constructor idiom.

```
struct Handle {
  enum Create { CREATE; };
  Handle(Create) { ... }
  Handle() { ... }
}

Handle h(Handle::CREATE)
```

Reply:
Neat, I didn't know about this one.